### PR TITLE
feat(neuron-wallet): push transaction history to neuron-ui

### PR DESCRIPTION
**Pull request by Keith-CY**
- push history on initialization, and monitoring TBD.
- `redirect` listener was added in `Router` in a callback, optimization for re-render will be done in another commit.

### CODE REVIEW

File: packages/neuron-wallet/src/main.ts [hunk 1]
```diff
@@ -20,6 +20,8 @@ const initUILayer = (win: BrowserWindow) => {
   })
   // switch language according to locale
   setLanguage(win, app.getLocale())
+  // send transaction history on window load
```
Comment (ashchan) [inline packages/neuron-wallet/src/main.ts]: Don't feel the comment is necessary. It just duplicates the function call - the function name already describes what it does well.

So is the switch language comment.

File: packages/neuron-wallet/src/channel.ts [hunk 2]
```diff
@@ -243,5 +248,19 @@ const listenToChannel = () => {
 export const setLanguage = (win: Electron.BrowserWindow, lng: string) => {
   win.webContents.send(Channel.SetLanguage, lng)
 }
+export const sendTransactions = (win: Electron.BrowserWindow, pageNo: number, pageSize: number) => {
```
Comment (ashchan) [inline packages/neuron-wallet/src/channel.ts]: The name `sendTransactions` gives us a impression that this is to send transactions to the node/blockchain.

Comment (Keith-CY) [reply packages/neuron-wallet/src/channel.ts]: renamed to `sendTransactionHistory`

File: packages/neuron-wallet/src/main.ts (hunk 1 above)
Comment (Keith-CY) [reply packages/neuron-wallet/src/main.ts]: comments removed

**Comment by Keith-CY** [comment]
updated, removed useless comments and rename `sendTransactions` to `sendTransactionHistory`

**Pull request merged**